src/Counter.php

```diff
@@ -41,16 +41,9 @@ final class Counter implements CounterInterface
 
     private int $ttlInSeconds = self::DEFAULT_TTL;
 
-    /**
-     * @var int Last increment time.
-     * In GCRA it's known as arrival time.
```

Comment:
This info was kinda important.

Reply:
Add comment to line 100, where created and used this variable.